src/components/NewCommentForm.jsx

```diff
@@ -0,0 +1,27 @@
+import { HiUser } from "react-icons/hi2";
+import { postNewCommentByArticleId } from "../../utils/utils"
+import '../css/new_comment_form.css'
+
+export default function NewCommentForm({article_id, newComment, setNewComment, listOfComments, setListOfComments}) {
+    const handleNewComment = (e) => {
+            e.preventDefault();
+            postNewCommentByArticleId(article_id, newComment)
+                .then(({comment}) => {
+                    setNewComment({username: "", body: ""});
+                    setListOfComments((listOfComments) => {
+                        return [comment, ...listOfComments]
+                    })
```

Comment:
Please include a .catch() block and let the user know if the posting was unsuccessful.